Source/cmake/GStreamerDefinitions.cmake

```diff
@@ -1,6 +1,7 @@
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_VIDEO PUBLIC ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEB_AUDIO PUBLIC ON)
 
+WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MANAGED_MEDIA_SOURCE PUBLIC ON)
```

Comment:
I don't think this option should be public unless you have a very strong reason for people to disable it...

Reply:
We want this option to be disabled by default for now. But compiled in so it can be enabled by a flag.

Ultimately, it's all very similar to ENABLE_MEDIA_SOURCE ; why would that one gets an option but not the other?

Reply:
I've removed this line.

Reply:
I think ENABLE_MEDIA_SOURCE is public because it originally required a newer version of GStreamer. Since that's no longer true, it no longer needs to be public (unless somebody has a real good use case for why MSE would need to be disabled at build time).

Reply:
Actually, this is the only place enabling that value (both ENABLE_MEDIA_SOURCE and ENABLE_MANAGED_MEDIA_SOURCE) so it's definitely required.

Would PRIVATE be more suitable here (I have no clue what those are for)

Reply:
PRIVATE is equivalent to CMake ADVANCED. It means the option is not exposed to end users (unless they peek behind the curtain). You need a very strong reason for it to be PUBLIC and I don't see one here, so it should be PRIVATE unless you have some justification for PUBLIC.

> Actually, this is the only place enabling that value (both ENABLE_MEDIA_SOURCE and ENABLE_MANAGED_MEDIA_SOURCE) so it's definitely required.

Well you originally said you want it disabled by default. If so, it must be OFF. Turning it ON means it's enabled by default. You can only choose one way or the other... which is it that you want? :P